README.md

```diff
@@ -15,7 +15,10 @@
         <img src="https://github.com/vapor/redis/actions/workflows/test.yml/badge.svg?event=push" alt="CI">
     </a>
     <a href="https://swift.org">
-        <img src="http://img.shields.io/badge/swift-5.5-brightgreen.svg" alt="Swift 5.5">
+        <img src="http://img.shields.io/badge/swift-5.6-brightgreen.svg" alt="Swift 5.6">
+    </a>
```

Comment:
the same comment as https://github.com/vapor/queues-redis-driver/pull/40#discussion_r1208108890